README.md

```diff
@@ -1,41 +1,41 @@
 # Introduction
 
-Welcome to the Universal Payload project. The goal of this project is to define an interface between a first stage platform initialization bootloader and a second stage payload.
+Welcome to the Universal Scalable Firmware project. The goal of this project is to define an interface between a first stage platform initialization bootloader and a second stage payload.
```

Comment:
Are we limiting the scope to Bootloader/Payload interface only?   I think the name is more broad.

Reply:
Yes. With new name, the scope should also be changed. Let me check to get the new scope and update this patch.